File: AISkinMeasurement/AISkinMeasurement/Modules/ViewController.m
//
//  ViewController.m
//  AISkinMeasurement
//
//  Created by Joe on 2020/5/18.
//  Copyright © 2020 Joe. All rights reserved.
//

#import "ViewController.h"
#import "SkinOverviewSection.h"
@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    self.datas = @[@"123",@"222"];
    
    [self.adater reloadDataWithCompletion:nil];
}

- (IGListSectionController *)listAdapter:(IGListAdapter *)listAdapter sectionControllerForObject:(NSArray *)object {
//    id element = object.firstObject;
    if ([object isKindOfClass:[NSString class]]) {
        return SkinOverviewSection.new;
    }else{
        return nil;
    }
}


@end

File: ARPracticeDemo/ARPracticeDemo/ViewController.swift
//
//  ViewController.swift
//  ARPracticeDemo
//
//  Created by Joe on 2020/4/28.
//  Copyright © 2020 Joe. All rights reserved.
//

import UIKit
class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        navigationController?.navigationBar.shadowImage = UIImage()
        navigationController?.navigationBar.setBackgroundImage(UIImage(), for: .default)
    }
    
}
